src/GitHub.VisualStudio/Views/GitHubPane/PullRequestCheckView.xaml

```diff
@@ -0,0 +1,60 @@
+﻿<local:GenericPullRequestCheckView
+            x:Class="GitHub.VisualStudio.Views.GitHubPane.PullRequestCheckView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:GitHub.VisualStudio.Views.GitHubPane"
+             xmlns:ghfvs="https://github.com/github/VisualStudio"
+             xmlns:c="clr-namespace:GitHub.VisualStudio.UI.Controls;assembly=GitHub.VisualStudio.UI"
+             xmlns:views="clr-namespace:GitHub.VisualStudio.Views"
+             xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI"
+             xmlns:vm="clr-namespace:GitHub.ViewModels.GitHubPane;assembly=GitHub.App"
+             xmlns:cache="clr-namespace:GitHub.UI.Helpers;assembly=GitHub.UI"
+             xmlns:sampleData="clr-namespace:GitHub.SampleData;assembly=GitHub.App"
+             xmlns:prop="clr-namespace:GitHub.VisualStudio.UI;assembly=GitHub.VisualStudio.UI"
+             xmlns:sampleData1="clr-namespace:GitHub.App.SampleData;assembly=GitHub.App"
+             mc:Ignorable="d" d:DesignWidth="600">
+
+    <d:DesignData.DataContext>
+        <sampleData1:PullRequestCheckViewModelDesigner />
```

Comment:
If namespace for this class is changed as per previous comment you can use `ghfvs:` XML namespace.